query-engine/query-engine-c-abi/README.md

```diff
@@ -0,0 +1,51 @@
+# Query Engine C (compatible) ABI
```

Comment:
Should we maybe explicitly mark this as "unstable" or similar?

Reply:
Yeah ofc